src/api/spec/models/bs_request_spec.rb

```diff
@@ -156,6 +156,17 @@
         expect(request.history_elements.first.type).to eq('HistoryElement::RequestDeleted')
       end
     end
+
+    context 'final state cannot be changed' do
+      let(:request) { create(:bs_request, state: 'accepted') }
+      let(:user) { create(:confirmed_user) }
```

Comment:
Can't you just the user created in line 9?

Reply:
@eduardoj yes and no. Yes I can, but actually i saw later that I cannot set the state in the creation, so I have to setup the factory further.. I refreshed my PR